src/foam/dao/JDBCConnectionSpec.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/* A JDBC Connection Specification to add to the system context */
+foam.CLASS({
+  package: 'foam.dao',
```

Comment:
Maybe a jdbc sub-package like foam.dao.jdbc instead. In time we could have a lot of JDBC related models and it would be better that they all be in one spot and not pollute the main dao package.

Reply:
OK